sdk/core/Azure.Core/src/WaitUntil.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure
```

Comment:
Curious: is it in the Azure namespace so it sits alongside `Operation<T>`?

Reply:
that was the idea yes